Gems/EMotionFX/Code/Source/Editor/Plugins/SkeletonOutliner/JointPropertyWidget.cpp

```diff
@@ -13,15 +13,33 @@
 #include <EMotionStudio/Plugins/StandardPlugins/Source/NodeWindow/NodeInfo.h>
 #include <EMotionStudio/Plugins/StandardPlugins/Source/NodeWindow/ActorInfo.h>
 #include <EMotionStudio/Plugins/StandardPlugins/Source/NodeWindow/NodeGroupInfo.h>
+#include <EMotionFX/CommandSystem/Source/RagdollCommands.h>
+#include <EMotionFX/CommandSystem/Source/CommandManager.h>
 #include <Editor/SkeletonModel.h>
 #include <Editor/InspectorBus.h>
+#include <Editor/ColliderHelpers.h>
 #include <UI/PropertyEditor/ReflectedPropertyEditor.hxx>
 #include <QItemSelectionModel>
 #include <QBoxLayout>
 #include <QLabel>
+#include <QHeaderView>
+#include <QLineEdit>
 
 namespace EMotionFX
 {
+    struct HLine : public QVBoxLayout
```

Comment:
suggestion: A comment describing why this type is needed and what it does would be helpful for other maintainers